unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DCLName : BaseDisposable
```

Comment:
Can we add a xml comment explaning why this component exists? I think a newcomer will scratch his head on this one.